source/write/update.txt

```diff
@@ -57,6 +57,8 @@ Each update function accepts the following parameters:
   and their usage, see the :manual:`Field Update Operators guide
   </reference/operator/update-field/>` in the {+mdb-server+} manual.
 
+- Options document: Specifies options to customize the operation, or ``NULL``.
```

Comment:
[s] This list isn't fully from the diff, but I think the [guidance around bolding terms](https://www.mongodb.com/docs/meta/style-guide/style/lists/#list-that-provides-terms-and-more-information) would apply her as well.